.github/workflows/pr-smoke-test.yml

```diff
@@ -96,7 +96,7 @@ jobs:
             await script({github, context})
 
       - name: Remove label from pull request.
-        if: "${{ contains(github.event.pull_request.labels.*.name, 'run: smoke tests') }}"
+        if: always()
```

Comment:
@MrJnrman Nice work. The reason for this condition was that the step fails with an error if the label is removed before this step but after the workflow started. If this action fails then the job is marked as failed. Can we revert this change so that we can try the other changes?